src/main/kotlin/no/nav/syfo/dialogmote/database/MoteQuery.kt

```diff
@@ -79,6 +78,24 @@ fun DatabaseInterface.getDialogmoteList(enhetNr: EnhetNr): List<PDialogmote> {
     }
 }
 
+const val queryGetDialogmoteUnfinishedListForEnhetNr =
+    """
+        SELECT *
+        FROM MOTE
+        INNER JOIN MOTEDELTAKER_ARBEIDSTAKER on MOTEDELTAKER_ARBEIDSTAKER.mote_id = MOTE.id
```

Comment:
Man trenger vel ikke gjøre denne join'en? (det gjelder nok queriet du tok utgangspunkt i). Tipper dette er blir kopiert fra der man henter møter for en arbeidstaker - og da må man jo joine.

Reply:
Sant! Jeg så den lå der, men glemte å fjerne. Fikser.